javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,15 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  // debugger;
+  var noStrings = [];
+  for (var i = 0; i < values.length; i++) {
+    // console.log('type of', typeof (values[i]));
+    if (typeof (values[i]) === 'number') {
```

Comment:
This function isn't supposed to collect all numbers, it's supposed to collect all values that are not strings.

Reply:
Just re-committed and pushed.

Thanks for the feedback!
<img width="448" alt="Screen Shot 2022-02-04 at 3 24 45 PM" src="https://user-images.githubusercontent.com/40590740/152616538-cc3dfaa8-2f3d-4f66-94bd-ea328c94bebc.png">